src/app/[locale]/viewer/_components/submodel-elements/SubmodelElementVisualizationProps.ts

```diff
@@ -0,0 +1,9 @@
+import { ISubmodelElement } from '@aas-core-works/aas-core3.0-typescript/types';
+
+
+export type SubmodelElementVisualizationProps = {
```

Comment:
Currently unused. We could also merge this structure with the CustomTreeItemContentProps, ContactInformationComponentProps and AddressComponentProps

Reply:
Agreed. I made them used at least once by the generic components. However, I did not merge them with CustomTreeContentProps, as I consider this out of the scope of this ticket, as this is already turning into a major refactoring.

Reply:
Yes you are right, I overlooked how big the TreeItemContentProps are